histomicstk/cli/NucleiDetection/__init__.py

```diff
@@ -0,0 +1,5 @@
+__all__ = (
+
+    # functions and classes of this package
+    'NucleiDetection'
+)
```

Comment:
Is there a reason you added this?  By default, an empty `__init__.py` should have exposed the other python files in that folder.

Reply:
This code was added during the debugging process. I will remove this now and convert the `__init__.py` into an empty file.